core/templates/dev/head/pages/exploration_editor/ExplorationDataService.js

```diff
@@ -60,7 +60,10 @@ oppia.factory('ExplorationDataService', [
       explorationId: explorationId,
       // Note that the changeList is the full changeList since the last
       // committed version (as opposed to the most recent autosave).
-      autosaveChangeList: function(changeList, successCallback, errorCallback) {
+      autosaveChangeList: function(
+          changeList,
+          successCallback = function(response) {},
```

Comment:
Is the unused arg necessary? (I.e. what happens if "response" is removed here?)

Reply:
It is necessary since when a call is made to successCallback [here](https://github.com/oppia/oppia/pull/6344/files/46c324c9aed6359b65151d776f27ba140d38e1d5#diff-f5cf086a2061475d6e1fc4cbdc574a4cL76), we pass in response as a parameter.

Reply:
So, just to confirm, removing it results in errors?

Reply:
Yes, it does (signature not matches error).